src/components/post/Post.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { CommentList } from '../CommentList/CommentList';
+import { ProtoPost } from '../ProtoPost';
+import { User } from '../User/User';
+
+export const Post = ({ post }) => (
+  <>
+    <h2>{post.title}</h2>
+    <p>
+      {post.body}
+    </p>
+    <User post={post} />
```

Comment:
You don't need to pass whole post `object` here, only `user`.
Also, spread operator here would help.